curriculum/challenges/english/25-front-end-development/workshop-tailwind-pricing-component/686386e76197458b7dfd52ef.md

```diff
@@ -0,0 +1,76 @@
+---
+id: 686386e76197458b7dfd52ef
+title: Step 6
+challengeType: 0
+dashedName: step-6
+---
+
+# --description--
+
+Now, outside the inner `div`, you need to create a CTA button. So, create an anchor element with an `href` of `#` and the text `Start listening`.
```

Comment:
it would be good to provide a short sentence explaining what a CTA is. The intro text says it is a call to action button but it would be better to explain exactly what a call to action is in a short sentence.